src/routes.ts

```diff
@@ -118,6 +119,22 @@ export default [
       ]
     }
   },
+  {
+    path: '/consent-amalgamation-out',
+    name: Routes.CONSENT_AMALGAMATION_OUT,
+    component: ConsentAmalgamationOut,
+    meta: {
+      requiresAuth: true,
+      breadcrumb: [
+        {
+          text: FilingNames.CONSENT_AMALGAMATION_OUT,
+          disabled: false,
+          exact: true,
+          to: { name: Routes.CONSENT_AMALGAMATION_OUT }
+        }
+      ]
+    }
+  },
```

Comment:
I didn't add a route for Amalgamation Out -- that can be done when we work on this feature.